tutorials/community/reinforcement_learning/README.md

```diff
@@ -0,0 +1,25 @@
+# Reinforcement Learning Protytopes on Vertex AI
```

Comment:
Spelling of "prototype" is wrong

Reply:
Thanks for catching this - This is addressed.